src/main/java/org/shanerx/tradeshop/TradeShop.java

```diff
@@ -94,47 +103,58 @@ public void onEnable() {
-		
-		for (String str : getConfig().getStringList("allowed-shops")) {
-			if (str.equalsIgnoreCase("shulker")) {
-				try {
-					inventories.addAll(Arrays.asList(Material.BLACK_SHULKER_BOX,
-							Material.BLUE_SHULKER_BOX,
-							Material.BROWN_SHULKER_BOX,
-							Material.CYAN_SHULKER_BOX,
-							Material.GRAY_SHULKER_BOX,
-							Material.GREEN_SHULKER_BOX,
-							Material.LIGHT_BLUE_SHULKER_BOX,
-							Material.LIME_SHULKER_BOX,
-							Material.MAGENTA_SHULKER_BOX,
-							Material.ORANGE_SHULKER_BOX,
-							Material.PINK_SHULKER_BOX,
-							Material.RED_SHULKER_BOX,
-							Material.SILVER_SHULKER_BOX,
-							Material.WHITE_SHULKER_BOX,
-							Material.YELLOW_SHULKER_BOX,
-							Material.PURPLE_SHULKER_BOX));
-				} catch (Throwable t) {
-					getLogger().info("You are on a version before 1.9, Shulker boxes are disabled!");
-				}
-			} else {
-				if (Material.valueOf(str) != null && allowedOld.contains(Material.valueOf(str)))
-					inventories.add(Material.valueOf(str));
-			}
-		}
-	}
+	private void addMaterials()
+    {
+        ArrayList<Material> AllowedOld = new ArrayList<>();
+        AllowedOld.addAll(Arrays.asList(new Material[] {Material.CHEST, Material.TRAPPED_CHEST, Material.DROPPER, Material.HOPPER, Material.DISPENSER}));
+
+        for(String str : getConfig().getStringList("allowed-shops"))
+        {
+            if(str.equalsIgnoreCase("shulker"))
+            {
+                try{
+                    inventories.addAll(Arrays.asList(new Material[] {Material.BLACK_SHULKER_BOX, 
+                            Material.BLUE_SHULKER_BOX, 
+                            Material.BROWN_SHULKER_BOX, 
+                            Material.CYAN_SHULKER_BOX, 
+                            Material.GRAY_SHULKER_BOX, 
+                            Material.GREEN_SHULKER_BOX, 
+                            Material.LIGHT_BLUE_SHULKER_BOX, 
+                            Material.LIME_SHULKER_BOX, 
+                            Material.MAGENTA_SHULKER_BOX, 
+                            Material.ORANGE_SHULKER_BOX, 
+                            Material.PINK_SHULKER_BOX, 
+                            Material.RED_SHULKER_BOX, 
+                            Material.SILVER_SHULKER_BOX, 
+                            Material.WHITE_SHULKER_BOX, 
+                            Material.YELLOW_SHULKER_BOX, 
+                            Material.PURPLE_SHULKER_BOX}));
+                } catch(Throwable t){
+                    getLogger().info(getName() + "You are on a version before 1.9, Shulkers are disabled!");
+                }
+            }
+            else
+            {
+                if(Material.valueOf(str) != null && AllowedOld.contains(Material.valueOf(str)))
+                    inventories.add(Material.valueOf(str));
+
+            }
+        }
+    }
+	
+	private void addDirections()
+    	{
+        ArrayList<Material> Allowed = new ArrayList<>();
```

Comment:
Please use lowerCamelCase for non constant variables.
(I'll change it this time.)